marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DbConfigsManager.java

```diff
@@ -0,0 +1,42 @@
+package com.marklogic.hub.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.extensions.ResourceManager;
+import com.marklogic.client.extensions.ResourceServices;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.util.RequestParameters;
+
+import java.io.IOException;
+import java.util.List;
+
+class DbConfigsManager extends ResourceManager {
+
+    private static final String NAME = "ml:dbConfigs";
+
+    private RequestParameters params = new RequestParameters();
+
+    public DbConfigsManager(DatabaseClient client) {
+        super();
+        client.init(NAME, this);
+    }
+
+    public ObjectNode generateIndexes(List<JsonNode> entities) {
```

Comment:
could it be better if we add throws exception in the method declaration...

Reply:
Yeah I left that in place, not knowing why it was there. Currently, if an error occurs, no exception is thrown, but rather an empty object node is returned. That doesn't break the client. I didn't want to change this to throw an error since it's out of scope of this JIRA ticket, and I didn't want to sort through the ramifications.